From 7d68b811ff2e1cfd98609aa36f4bc1684d483d00 Mon Sep 17 00:00:00 2001
From: Xi Ruoyao <xry111@xry111.site>
Date: Sat, 12 Jul 2025 19:38:09 +0800
Subject: [PATCH 61/67] AOSCOS: net/phytmac: Remove unnecessary
 pcim_iounmap_regions() call

pcim_iounmap_regions() is removed in commit 855c634930f0 ("PCI: Remove
pcim_iounmap_regions()").  Moreover, it is not necessary to call it in
the driver's remove() function, since it does cleanup automatically on
driver detach.

Signed-off-by: Xi Ruoyao <xry111@xry111.site>

Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>
---
 drivers/net/ethernet/phytium/phytmac_pci.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/net/ethernet/phytium/phytmac_pci.c b/drivers/net/ethernet/phytium/phytmac_pci.c
index fac5d717ad76..69f34c2faaf5 100644
--- a/drivers/net/ethernet/phytium/phytmac_pci.c
+++ b/drivers/net/ethernet/phytium/phytmac_pci.c
@@ -177,7 +177,6 @@ static void phytmac_pci_remove(struct pci_dev *pdev)

 	phytmac_free_pdata(pdata);
 	bar_mask = pci_select_bars(pdev, IORESOURCE_MEM);
-	pcim_iounmap_regions(pdev, bar_mask);

 	pci_disable_device(pdev);
 }
--
2.50.1
